client/src/main/com/sinch/sdk/domains/conversation/api/v1/adapters/messages/OmniMessageOverrideMapperHelper.java

```diff
@@ -57,19 +55,16 @@ public static void serialize(OmniMessageOverride value, JsonGenerator jgen) thro
     } else if (value instanceof ListMessage) {
       field = ListMessageFieldInternal.builder().setListMessage((ListMessage) value).build();
     } else if (value instanceof LocationMessage) {
-      field =
-          LocationMessageFieldInternal.builder()
-              .setLocationMessage((LocationMessage) value)
-              .build();
+      field = value;
```

Comment:
Is is also related to generator delegation feature: not all fields are moved to use this feature.
Here Location and Media are using it: so when instancez of this classes is having to be serialized; it will be "automatic".
Others are still "old" way managed.
Because of webhooks schemas/taxonomy had already  "spread" out of the the dedicated "webhooks" field; moving fields to support delegation was not added to this PR (but will have to come before releasing the final feature)